samples/bluetooth/peripheral/prj.conf

```diff
@@ -4,6 +4,6 @@ CONFIG_BLUETOOTH_SMP=y
 CONFIG_BLUETOOTH_SIGNING=y
 CONFIG_BLUETOOTH_PERIPHERAL=y
 CONFIG_BLUETOOTH_ATT_PREPARE_COUNT=2
-CONFIG_BLUETOOTH_GATT_DYNAMIC_DB=y
 CONFIG_BLUETOOTH_PRIVACY=y
 CONFIG_BLUETOOTH_DEVICE_NAME="Test peripheral"
+CONFIG_BLUETOOTH_DEVICE_APPEARANCE=0x0341
```

Comment:
Since the assigned numbers use decimal integers I'd say your Kconfig definition is ok, and the hex versions should simply be converted to decimal when moved from the code to the prj.conf.